src/routes/transactions/__tests__/controllers/list-multisig-transactions-by-safe.transactions.controller.spec.ts

```diff
@@ -234,29 +340,51 @@ describe('List multisig transactions by Safe - Transactions Controller (Unit)',
       .get(`/v1/chains/${chainId}/safes/${safeAddress}/multisig-transactions`)
       .expect(200)
       .then(({ body }) => {
-        expect(body).toEqual(
-          getJsonResource(
-            'multisig-transactions/erc721/expected-response.json',
-          ),
-        );
+        expect(body).toMatchObject({
+          results: [
+            {
+              type: 'TRANSACTION',
+              transaction: {
+                id: 'multisig_0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C_0x0f9f1b72',
+                timestamp: 1655853152000,
+                txStatus: 'SUCCESS',
+                txInfo: {
+                  type: 'Transfer',
+                  sender: { value: safe.address },
+                  recipient: {
+                    value: '0x3a55e304D9cF13E45Ead6BA3DabCcadD3a419356',
+                  },
+                  direction: 'OUTGOING',
+                  transferInfo: {
+                    type: 'ERC721',
+                    tokenAddress: token.address,
+                    tokenId: '495',
+                    tokenName: token.name,
+                    tokenSymbol: token.symbol,
+                    logoUri: token.logoUri,
+                  },
+                },
+                executionInfo: {
+                  type: 'MULTISIG',
+                  nonce: multisigTransaction.nonce,
+                  confirmationsRequired: 3,
+                  confirmationsSubmitted: 3,
+                  missingSigners: null,
+                },
+                safeAppInfo: null,
+              },
+              conflictType: 'None',
+            },
+          ],
+        });
       });
   });
 
   it('Should get a Custom transaction mapped to the expected format', async () => {
     const chainId = faker.string.numeric();
     const chainResponse = chainBuilder().build();
```

Comment:
```suggestion
    const chainResponse = chainBuilder().with('chainId', chainId).build();
```

Reply:
Changed in [0450df9](https://github.com/safe-global/safe-client-gateway-nest/pull/503/commits/0450df9f6fa71b7b2f5bd8b364e0fa0aedbdb2f3)